classes/Entities/BadgeIssued.php

```diff
@@ -20,4 +20,12 @@ class BadgeIssued extends BaseModel
      * @var array
      */
     protected $guarded = ['id'];
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\belongsTo
+     */
+    public function badges()
```

Comment:
As below, this returns on badge as per the defined relationships so should be named as such.

Reply:
Done

Reply:
YOu just need to rename the function to badge() to reflect the fact that it is a singular relationship

Reply:
Yes, I did change to badge()